packages/teleport/src/Main/useMain.ts

```diff
@@ -16,11 +16,14 @@ limitations under the License.
 
 import { useState } from 'react';
 import useAttempt from 'shared/hooks/useAttemptNext';
-import useTeleport from 'teleport/useTeleport';
 import { Feature } from 'teleport/types';
+import teleportContext from 'teleport/teleportContext';
+import teleportEContext from 'e-teleport/teleportEContext';
```

Comment:
> import teleportEContext from 'e-teleport/teleportEContext';

devs wont be able to build the OSS version. OSS should not have _any_ references to E.

Reply:
ah got it